client/src/api/profile.ts

```diff
@@ -0,0 +1,20 @@
+import type { UserProfile } from "../types";
+import { getToken } from "../token";
+
+export const editProfile = async (profileBody: UserProfile) => {
```

Comment:
This functions returns a type of `any`.  Can you parse the response and return a `UserProfile`? We should keeps things typesafe.

Reply:
Will do 👍